page/page.js

```diff
@@ -4,6 +4,11 @@ function init (app) {
   app.get('/example', function (req, res) {
     require('./example').page(req, res)
   })
+
+  app.get('/inputvalidator', function (req, res) {
```

Comment:
Please change the name to `input_validator`. This applies to all files/names in this pr